src/storage/v2/property_store.cpp

```diff
@@ -1292,12 +1292,11 @@ void PropertyStore::SetBuffer(const std::string_view buffer) {
 
   uint64_t size = 0;
   uint8_t *data = nullptr;
+  size = buffer.size();
```

Comment:
Good catch! Does it mean that in loop on 1304 line we lost last byte value?

Reply:
Yes and that was edge case :bug: :lady_beetle: